config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
Sorry `(boolean)` cast doesn't work. The ternary that @ionas showed might work, but it doesn't work if you set "True". Maybe the following?

```php
<?php

$debug = getenv('DEBUG');

var_dump($debug);

$debug = filter_var($debug, FILTER_VALIDATE_BOOLEAN);

var_dump($debug);

if ($debug) {
    echo "No---\n";
}
```